atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -259,7 +259,8 @@ private TableReference getTableToSweep(SweepTransaction t,
     }
 
     private double getSweepPriority(SweepPriorityRowResult oldPriority, SweepPriorityRowResult newPriority) {
-        if (AtlasDbConstants.hiddenTables.contains(newPriority.getRowName().getFullTableName())) {
+        Set<String> hiddenTableFullNames = AtlasDbConstants.hiddenTables.stream().map(tableRef -> tableRef.getQualifiedName()).collect(Collectors.toSet());
```

Comment:
If you want to kill time, the thread on deprecating `java.util.Optional#get()` is sad example of foot Kevlar: http://mail.openjdk.java.net/pipermail/core-libs-dev/2016-April/040484.html